src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { SelectOption } from '../SelectOption/SelectOption';
+import { TextField } from '../TextField/TextField';
+
+interface Props {
+  onChangeTodo: (title: string, userId: number, completed?: boolean) => void;
+}
+
+export const NewTodo: React.FC<Props> = ({ onChangeTodo }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number>(0);
+  const [submited, setSubmited] = useState<boolean>(false);
+
+  const hasErrorTextField = title.trim() === '' && submited;
+  const hasErrorSelectOption = selectedUser === 0 && submited;
+
+  function resetForm() {
+    setSelectedUser(0);
+    setTitle('');
+    setCount(cur => cur + 1);
+    setSubmited(false);
+  }
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+
+    if (selectedUser === 0 || title.trim() === '') {
```

Comment:
change and use it in all such cases
```suggestion
    if (!selectedUser || !title.trim().length) {
```